src/api.js

```diff
@@ -286,16 +286,7 @@ api.declare({
   output:         'list-tasks-response.json#',
   title:          'List Tasks',
```

Comment:
Please add `noPublish: true` to avoid publishing this end-point in references...